ndk/src/bitmap.rs

```diff
@@ -26,10 +26,14 @@ pub enum BitmapError {
     BadParameter = ffi::ANDROID_BITMAP_RESULT_BAD_PARAMETER,
     #[doc(alias = "ANDROID_BITMAP_RESULT_JNI_EXCEPTION")]
     JniException = ffi::ANDROID_BITMAP_RESULT_JNI_EXCEPTION,
-    // Use the OK discriminant, as no-one will be able to call `as i32` and only has access to the
-    // constants via `From` provided by `IntoPrimitive` which reads the contained value.
+
+    // Use the SUCCESS discriminant, as no-one will be able to call `as i32` and only has access to
+    // the constants via `From` provided by `IntoPrimitive` which reads the contained value.
+    // An autogenerated `<previous variant> + 1` discrriminant is normally fine, except that the
```

Comment:
```suggestion
    // An autogenerated `<previous variant> + 1` discriminant is normally fine, except that the
```

Reply:
Found the same c-p error in `media_error.rs` :)